contrib/inventory/foreman.ini

```diff
@@ -123,13 +123,45 @@ user = foreman
 password = secret
 ssl_verify = True
 
+#Foreman 1.24 introduces a new reports API to improve performance of the inventory script. Set to False if you want to use the old API. Defaults to True.
+
+use_reports_api = True
+
 # Retrieve only hosts from the organization "Web Engineering".
 # host_filters = organization="Web Engineering"
 
 # Retrieve only hosts from the organization "Web Engineering" that are
 # also in the host collection "Apache Servers".
 # host_filters = organization="Web Engineering" and host_collection="Apache Servers"
 
+# Foreman Inventory report related configuration options.
+# Configs that default to True :
+# want_organization , want_location, want_IPv4, want_host_group, want_subnet, want_smart_proxies, want_facts
+# Configs that default to False :
+# want_IPv6, want_subnet_v6, want_content_facet_attributes, want_host_params
+
+[report]
+# want_organization = True
+# want_location = True
+# want_IPv4 = True
+# want_IPv6 = False
```

Comment:
Personally I consider IPv4 legacy and IPv6 default. Can we enable this by default?

Reply:
It's not fully supported in Foreman, so it make sense to default to ipv4 to me. Also these values reflect the default in the inventory template on Foreman side, I'd start with the change there.

Reply:
I agree it should be changed in the template as well. To be clear: I'd expect dual stack support by default so both IPv4 and IPv6 should default to true IMHO.